src/modules/universal-discovery/universal.discovery.module.tsx

```diff
@@ -223,7 +226,7 @@ export default class UniversalDiscoveryModule extends Component {
     findContentType(identifier) {
         let contentType = null;
 
-        Object.values(this.props.contentTypes).forEach((group) => {
+        Object.values(this.props.contentTypes).forEach((group: any) => {
```

Comment:
At the beginning some type errors can be fixed by assigning `any` type, which then can be converted into something more specific in the future. As we will be adding more and more types.